### ISSUE TITLE
Error messages in Qucslib

### ISSUE THREAD

**Issue by ra3xdh** [Qucs/qucs#591]
Every user library with `Default symbol` defined gives an error message when trying to open in library manager. Steps to reproduce:

* Move `NMOSFETs.lib` library into `$HOME/.qucs/user_lib`
* Select this library at ComboBox. Use user version of library.
* The following error message will be shown:
![liberror](https://cloud.githubusercontent.com/assets/4920080/20036893/cc3e9518-a42e-11e6-969f-d8e205e9c6a3.png)

It coul d be seen from the error text that library file name is not determined properly.  I think, it appears after #556

**Comment by felix-salfelder** [comment]
FTR: i have rebased the non-hotfix approach to #556. see #592. (help definitely appreciated).

**Comment by ra3xdh** [comment]
#592 is not a good solution to fix this particular issue, because it is a deep refactoring proposal. I don't recommend start refactoring right at release time. Let's discuss your refactoring concepts after 0.0.19 will be released.

**Comment by felix-salfelder** [comment]
yes, the non-hotfix approach is currently targeting `devel`, only with some help from others that might change.

and no, there's no "deep refactoring" involved just for this, i intentionally postponed that. read the comments/commits and disussion.

**Comment by ra3xdh** [comment]
There exists another issue in Library Manager. It always show an an empty symbol for devices (not subcircuits) libraries. Try to select `Transistor` or `Diode` library and you will see the following. Component symbol will be empty.
![liberror1](https://cloud.githubusercontent.com/assets/4920080/20218836/e6b6699c-a83f-11e6-9c19-ddf38d8b4ab6.png)

**Comment by guitorri** [comment]
Seems fixed with #594
